### ISSUE TITLE
trend_line_parameter_request

### ISSUE THREAD

**Issue by Evadzi** [CCI-Tools/cate#692]
@JanisGailis 
Is it possible to have one additional  parameter for the 'plot' operation to display trend (line and value) for time series plot?

this parameter should also be created/enabled for the 'create time series plot from selected placemark'  feature for the highlighted variable in the Cate's Workspace area.

**Comment by JanisGailis** [comment]
Do you mean doing a linear regression, or just adding a line ``y = ax + b``, where `a` and `b` can be defined? What do you mean by trend value?

I think it's possible only to add this for the time-series plot, as ``plot()`` is a very generic operation that works on all kinds of data.

**Comment by Evadzi** [comment]
@JanisGailis 
Thank you. yes for now the line y=ax+b to fine that best fits a data set in the least squares sense is okay.
In the future with regards to the plot() operator, Is it possible to have option for multiple plots?

**Comment by JanisGailis** [comment]
That is a question for @forman. If I remember correctly he researched that option and it was no trivial (impossible) to implement with the current plotting library.

**Comment by forman** [comment]
Yes, that's a very reasonable if not mandatory requirement for any data analysis. It should become part at least of the scatter plot. I don't remember the actual problem at that time anymore, but it should be easy to do in matplotlib, [see here](http://widu.tumblr.com/post/43624347354/matplotlib-trendline) for example.
